src/client/telemetry/index.ts

```diff
@@ -811,6 +811,7 @@ export interface IEventNamePropertyMapping {
             | 'failed' // Installation disabled (this is what python extension returns).
             | 'install' // User chose install from prompt.
             | 'donotinstall' // User chose not to install from prompt.
+            | 'differentKernel' // User chose not to select a different kernel/interpreter.
```

Comment:
I'm confused, didn't you change the wording here?

Reply:
Damn it, i thought it did, then made another change to that, & lost it.

Reply:
Wil update here and in main